### PR TITLE
Refactor function signatures of data client create and delete operations

### PR DESCRIPTION
**Pull request by Chuhaa**
# Description
Refactor function signature

Fixes # ([issue](https://github.com/ballerina-platform/ballerina-extended-library/issues/201))

One line release note: 
- One line describing the feature/improvement/fix made by this PR 

## Type of change

Please delete options that are not relevant.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

**Test Configuration**:
* Ballerina Version: 2201.0.0
* Operating System: Ubuntu
* Java SDK: JDK 11

# Checklist:

### Security checks
 - [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by LakshanSS — COMMENTED** [review]
Please do a major version bump as discussed offline.
(Since we are introducing breaking changes)

File: cosmosdb/Ballerina.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 distribution = "2201.0.0"
 org= "ballerinax"
 name= "azure_cosmosdb"
-version= "2.2.0"
+version= "3.0.0"
```
Comment (LakshanSS) [inline cosmosdb/Ballerina.toml:5]: This version change will happen in Depenedencies.toml also.
Please commit that change also

Comment (Chuhaa) [reply cosmosdb/Ballerina.toml:5]: Updated

**Comment by Chuhaa** [comment]
> Please do a major version bump as discussed offline. (Since we are introducing breaking changes)

Updated to 3.0.0

**Review by LakshanSS — APPROVED** [review]

**Comment by abeykoon** [comment]
Does this address https://github.com/wso2-enterprise/choreo/issues/7295?

**Review by sachinira — COMMENTED** [review]
Please update the samples and documentations according to the new revamp

**Comment by Chuhaa** [comment]
> Please update the samples and documentations according to the new revamp

Updated in https://github.com/ballerina-platform/module-ballerinax-azure-cosmosdb/pull/59/commits/bff2eb449e7ada3faf70379a51fe6a8931aec8ee

File: cosmosdb/Module.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ designed to store and query JSON-like documents. Therefore, the document you cre
     int valueOfPartitionKey = 0;
     string id = "my_document";
 
-    check azureCosmosClient-> createDocument("my_database", "my_container", id, document, valueOfPartitionKey);
+    cosmosdb:DocumentResponse response = check azureCosmosClient-> createDocument("my_database", "my_container", id, document, valueOfPartitionKey);
     ```
 **Note:** <br/>
 - This document is created inside an already existing container with ID **my_container** and the container was created inside a database with ID **my_document**.
```
Comment (sachinira) [inline cosmosdb/Module.md:60]: Update the database ID to "my_database". This was an error from the beginning. Users who are new to cosmos DB sometimes gets confused by it :)

Comment (Chuhaa) [reply cosmosdb/Module.md]: Updated in my_database

**Review by sachinira — APPROVED** [review]

**Pull request merged**